### PR TITLE
Russian translation: global tool configuration, new node

### PR DESCRIPTION
**Pull request by Riliane**
Translation of items on the global tool configuration, node monitors, new node screens. Includes fixing some typos and out of context mistranslations and a bit of unification of the terms used, updating an outdated help blurb, un-hardcoding some validation fail messages.

### Proposed changelog entries

- Add Russian translation of text on the global tool configuration, node monitors, new node screens page.

### Proposed upgrade guidelines

N/A

### Submitter checklist

- [N/A] The Jira issue, if it exists, is well-described.
- [x] The changelog entries and upgrade guidelines are appropriate for the audience affected by the change (users or developers, depending on the change) and are in the imperative mood (see [examples](https://github.com/jenkins-infra/jenkins.io/blob/master/content/_data/changelogs/weekly.yml)).
  - Fill in the **Proposed upgrade guidelines** section only if there are breaking changes or changes that may require extra steps from users during upgrade.
- [N/A] There is automated testing or an explanation as to why this change has no tests.
- [N/A] New public classes, fields, and methods are annotated with `@Restricted` or have `@since TODO` Javadocs, as appropriate.
- [N/A] New deprecations are annotated with `@Deprecated(since = "TODO")` or `@Deprecated(forRemoval = true, since = "TODO")`, if applicable.
- [N/A] New or substantially changed JavaScript is not defined inline and does not call `eval` to ease future introduction of Content Security Policy (CSP) directives (see [documentation](https://www.jenkins.io/doc/developer/security/csp/)).
- [N/A] For dependency updates, there are links to external changelogs and, if possible, full differentials.
- [N/A] For new APIs and extension points, there is a link to at least one consumer.

### Desired reviewers

@yaroslavafenkin @Pldi23 @duemir 

### Maintainer checklist

Before the changes are marked as `ready-for-merge`:

- [ ] There are at least two (2) approvals for the pull request and no outstanding requests for change.
- [ ] Conversations in the pull request are over, or it is explicit that a reviewer is not blocking the change.
- [ ] Changelog entries in the pull request title and/or **Proposed changelog entries** are accurate, human-readable, and in the imperative mood.
- [ ] Proper changelog labels are set so that the changelog can be generated automatically.
- [ ] If the change needs additional upgrade steps from users, the `upgrade-guide-needed` label is set and there is a **Proposed upgrade guidelines** section in the pull request title (see [example](https://github.com/jenkinsci/jenkins/pull/4387)).
- [ ] If it would make sense to backport the change to LTS, a Jira issue must exist, be a _Bug_ or _Improvement_, and be labeled as `lts-candidate` to be considered (see [query](https://issues.jenkins.io/issues/?filter=12146)).

<a href="https://gitpod.io/#https://github.com/jenkinsci/jenkins/pull/7321"></a>

### CODE REVIEW

**Review by Wadeck — COMMENTED** [review]
Great, thanks especially for the i18n-ing of some fixed Strings :)

(no idea about the content itself ;) )

File: core/src/main/resources/jenkins/tools/GlobalToolConfiguration/index_ru.properties
Comment (Riliane) [reply core/src/main/resources/jenkins/tools/GlobalToolConfiguration/index_ru.properties:1]: there are actually a bunch of disordered translations for "dashboard". "Вернуться к главной странице" which i found and duplicated, "На главную", "Домой". "Dashboard" itself is "ИнфоПанель" in one place. tbh, it's kind of a mess and once again we should probably stick to one and unify

File: core/src/main/resources/hudson/slaves/DumbSlave/newInstanceDetail_ru.properties
Comment (Riliane) [reply core/src/main/resources/hudson/slaves/DumbSlave/newInstanceDetail_ru.properties:27]: > Select this type if no other agent types apply — for example such as when you are adding \
  a physical computer, virtual machines managed outside Jenkins, etc.

I think there's a bit an emphasis on how this is the "last resort" agent type, the most generic options (better options being provided possibly by plugins).

File: core/src/main/resources/hudson/security/GlobalSecurityConfiguration/config_ru.properties
Comment (Riliane) [reply core/src/main/resources/hudson/security/GlobalSecurityConfiguration/config_ru.properties:30]: then maybe "Установки JDK" to match?

File: core/src/main/resources/hudson/model/Messages_ru.properties
Comment (Pldi23) [inline core/src/main/resources/hudson/model/Messages_ru.properties:104]:
```suggestion
Slave.InvalidConfig.NoName=Некорректная настройка агента. Имя отсутствует
```

File: core/src/main/resources/hudson/model/Messages_ru.properties
Comment (Pldi23) [inline core/src/main/resources/hudson/model/Messages_ru.properties:106]:
```suggestion
  в качестве корня ФС? Обратите внимание, контроллеру необязательно иметь доступ к указанной директории.
```

File: core/src/main/resources/hudson/slaves/JNLPLauncher/main_ru.properties
Comment (Pldi23) [inline core/src/main/resources/hudson/slaves/JNLPLauncher/main_ru.properties:25]: Just to double check, don't we need upper-cased first latter?

File: core/src/main/resources/hudson/tasks/Messages_ru.properties
Comment (Pldi23) [inline core/src/main/resources/hudson/tasks/Messages_ru.properties:59]: not sure, maybe 'Установить с помощью Apache' would slightly better describe the sense.

**Comment by Riliane** [comment]
whoops - somehow when applying patches I lost the actual values of the strings I unhardcoded from the commit. they're back now.

File: core/src/main/resources/hudson/slaves/JNLPLauncher/main_ru.properties
Comment (Riliane) [reply core/src/main/resources/hudson/slaves/JNLPLauncher/main_ru.properties:25]: it's lowercase in every other language so probably a link in the middle of text - but I tried looking for it and actually it might just be unused now?

File: core/src/main/resources/hudson/tasks/Messages_ru.properties
Comment (Riliane) [reply core/src/main/resources/hudson/tasks/Messages_ru.properties:59]: maybe something like "из репозитория Apache"?

Comment (Pldi23) [reply core/src/main/resources/hudson/tasks/Messages_ru.properties:59]: yes, I like "Установить из репозитория Apache"

File: core/src/main/resources/hudson/slaves/JNLPLauncher/main_ru.properties
Comment (daniel-beck) [reply core/src/main/resources/hudson/slaves/JNLPLauncher/main_ru.properties:25]: Looks like a leftover translation from https://github.com/jenkinsci/jenkins/pull/6543/files#diff-731cfabfa254305495da8a5d54f327e47d6229868df1c286b644500a998faa85L44

File: core/src/main/resources/hudson/security/GlobalSecurityConfiguration/config_ru.properties
Comment (yaroslavafenkin) [reply core/src/main/resources/hudson/security/GlobalSecurityConfiguration/config_ru.properties:30]: Yeah sounds good, somehow I've missed that

**Review by oleg-nenashev — APPROVED** [review]
Just a few suggestions. Looks good otherwise

File: core/src/main/resources/hudson/model/Slave/help-remoteFS_ru.html
Comment (Riliane) [reply core/src/main/resources/hudson/model/Slave/help-remoteFS_ru.html:28]: I have capitalized it to indicate it's the name of another option, but I guess it's more of a thing in English 🤔

**Review by Pldi23 — APPROVED** [review]

File: core/src/main/resources/hudson/slaves/JNLPLauncher/config_ru.properties
Comment (Riliane) [reply core/src/main/resources/hudson/slaves/JNLPLauncher/config_ru.properties:1]: tried to look for this in the UI if there's a help blurb with more context, but it seems undisplayed. is this property not an actually displayed rendered element?
```
<f:property title="${%Enable work directory}" field="workDirSettings" />
```

File: core/src/main/resources/hudson/slaves/DumbSlave/newInstanceDetail_ru.properties
Comment (yaroslavafenkin) [reply core/src/main/resources/hudson/slaves/DumbSlave/newInstanceDetail_ru.properties:3]: Should be addressed I think

File: core/src/main/resources/hudson/slaves/DumbSlave/newInstanceDetail_ru.properties
Comment (yaroslavafenkin) [reply core/src/main/resources/hudson/slaves/DumbSlave/newInstanceDetail_ru.properties:27]: Then how about `Этот тип агента, вероятно, подойдет если никакой другой не подходит - например...`?

I kinda dislike how current version sounds.

Comment (Riliane) [reply core/src/main/resources/hudson/slaves/DumbSlave/newInstanceDetail_ru.properties:27]: maybe at least "Этот тип агента стоит выбрать, если никакой другой не подходит"? I kind of dislike the hesitation and uncertainty in your suggestion, like it only _might_ work. there wasn't any in the English version: `Select this type if no other agent types apply`

File: core/src/main/resources/hudson/security/GlobalSecurityConfiguration/config_ru.properties
Comment (Riliane) [reply core/src/main/resources/hudson/security/GlobalSecurityConfiguration/config_ru.properties:29]: We could switch the "инсталляция Ant" to "установка Ant" to unify, if this is the better direction

File: core/src/main/resources/hudson/slaves/DumbSlave/newInstanceDetail_ru.properties
Comment (yaroslavafenkin) [reply core/src/main/resources/hudson/slaves/DumbSlave/newInstanceDetail_ru.properties:27]: `вероятно` can be dropped. I thought there was uncertainty there so added it.
Your version is good for me too.

File: core/src/main/resources/jenkins/tools/GlobalToolConfiguration/index_ru.properties
Comment (yaroslavafenkin) [reply core/src/main/resources/jenkins/tools/GlobalToolConfiguration/index_ru.properties:1]: I was leaning towards `панель управления` but now that I think of it it seems different (`Control panel` in Windows for example). Guess that makes `панель задач` a better option.
Maybe it can be left out to not block this PR and unified as a separate PR.

Comment (Riliane) [reply core/src/main/resources/jenkins/tools/GlobalToolConfiguration/index_ru.properties:1]: "панель задач" also makes me kind of think of windows taskbar, so I guess it's on the same level as "панель управления", with a potential alternate meaning.

File: core/src/main/resources/hudson/security/GlobalSecurityConfiguration/config_ru.properties
Comment (Riliane) [reply core/src/main/resources/hudson/security/GlobalSecurityConfiguration/config_ru.properties:29]: switching to "установка" at least for now. better to have unity.

**Review by yaroslavafenkin — APPROVED** [review]

**Comment by NotMyFault** [comment]
/label ready-for-merge

---

This PR is now ready for merge. We will merge it after ~24 hours if there is no negative feedback.
Please see the [merge process documentation](https://github.com/jenkinsci/jenkins/blob/master/docs/MAINTAINERS.adoc#merge-process) for more information about the merge process.
Thanks!

**Review by duemir — APPROVED** [review]

**Pull request merged**
